composer.json

```diff
@@ -6,6 +6,7 @@
 	],
 	"require": {
 		"php": "^8.0",
+		"ext-json": "*",
```

Comment:
This isn't needed in composer.json, please remove it.

Reply:
How should I fix the missing extension failing CI then?

https://github.com/phpstan/phpstan-src/runs/5677868785?check_suite_focus=true

![image](https://user-images.githubusercontent.com/924196/159937953-3f164344-eb00-44dd-b645-cdc35da50b74.png)

Reply:
Thx